pcdet/utils/grid_utils.py

```diff
@@ -0,0 +1,19 @@
+import torch
```

Comment:
Maybe it is no need to create a separate file for this simple function. 
Such as we could merge `grid_utils.py`, `depth_utils.py` and `transform_utils.py` as a single file `transform_utils.py`.

Reply:
Done